templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -615,9 +616,9 @@ Mappings:
       MapName: TimeoutsSingleCluster
   TimeoutsSingleCluster:
     'No': #! single cluster deployment without relocation
-      CloudInit:          "1500" # 25m
-      TAPInstall:          "600" # 10m
-      TAPWorkloadInstall:  "600" # 10m
+      CloudInit:          "3300" # 55m
+      TAPInstall:         "2100" # 35m
+      TAPWorkloadInstall:  "900" # 15m
```

Comment:
The present timeout value has issues, Hence I have to bump it up. I did not want to be conservative as this is just a timeout value and I happen to use the same as in multi-cluster. I know this might be a little longer for this use case but we can fine-tune it if we got more time.